lib/alchemy/test_support/factories/cell_factory.rb

```diff
@@ -4,7 +4,7 @@
 FactoryGirl.define do
 
   factory :alchemy_cell, class: 'Alchemy::Cell' do
-    page { Alchemy::Page.find_by(language_root: true) || FactoryGirl.create(:alchemy_page, :language_root) }
+    page do Alchemy::Page.find_by(language_root: true) || FactoryGirl.create(:alchemy_page, :language_root) end
```

Comment:
Metrics/LineLength: Line is too long. [111/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)